src/wasm-builder.h

```diff
@@ -35,10 +35,11 @@ struct NameType {
 
 class Builder {
   MixedArena& allocator;
+  bool is64;
```

Comment:
All callers have a valid module, so wasn't necessary to make it a `*`

Reply:
This is going to be inconsistent with Poppy IR, where individual functions rather than the module are the source of truth for the extra validation context. I would have preferred taking the risk of stale data. Most builders are not long-lived, anyhow, and most passes will not be changing the validation context in any way. I just uploaded #3144, but I would actually prefer to keep the allocator and use an `is64` bool. @kripken, wdyt?

Reply:
I'm confused, why would Poppy IR change the relationship of functions to the module?

Anyhow it's late on Friday, let's discuss this in depth on Monday?